pkg/chart/metadata.go

```diff
@@ -112,7 +112,7 @@ func (md *Metadata) Validate() error {
 		return ValidationError("chart.metadata.name is required")
 	}
 
-	if md.Name != filepath.Base(md.Name) {
+	if !filepath.IsLocal(md.Name) {
```

Comment:
The [documentation has some details on what is allowed and not allowed in a name](https://helm.sh/docs/chart_best_practices/conventions/#chart-names). For example, a `.` is not allowed in a name. But, `filepath.IsLocal` returns true when a `.` is in the name. So, it's not good to use for this kind of validation.